tests/unittests/graphTest.cpp

```diff
@@ -299,3 +299,40 @@ TEST(Graph, nodesWithPredicates) {
   EE.compile(CompilationMode::Infer, F);
   EE.run({input}, {&inputs});
 }
+
+// Return the number of ConvolutionNode after lower.
+int getConvNodeSize(BackendKind kind) {
+  Module mod;
+  Function *F = mod.createFunction("main");
+  IRFunction M(F);
+  auto *input = mod.createVariable(ElemKind::FloatTy, {1, 2, 1, 32}, "input");
+  ConvolutionNode *CN = F->createConv("conv", input, 6, 1, 1, 0, 2);
+  F->createSave("save", CN);
+
+  std::unique_ptr<Backend> backend_;
+  backend_.reset(createBackend(kind, &M));
+  lower(F, CompilationMode::Infer, *backend_);
+
+  unsigned count = 0;
+  for (auto *n : F->getNodes()) {
+    if (n->getKind() == Kinded::Kind::ConvolutionNodeKind) {
+      count++;
+    }
+  }
+
+  if (kind == BackendKind::CPU)
+    EXPECT_EQ(count, 1);
```

Comment:
would be good to use {} consistently. Like having {} around Expect_eq, similar to line 318.